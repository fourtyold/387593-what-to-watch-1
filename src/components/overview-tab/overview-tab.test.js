import React from "react";
import renderer from "react-test-renderer";

import OverviewTab from "./overview-tab.jsx";

const options = {
  film: {
    director: `director`,
    rating: 5,
    scoresCount: 10000,
    starring: [`actor1`, `actor2`, `actor3`]
  }
};

it(`Overview tab correctly renders`, () => {
  const tree = renderer.create(
      <OverviewTab
        film={options.film}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
