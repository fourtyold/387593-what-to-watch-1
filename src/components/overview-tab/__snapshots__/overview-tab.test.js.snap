// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview tab correctly renders 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      5
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        good
      </span>
      <span
        className="movie-rating__count"
      >
        10000
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        director
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        actor1, actor2, actor3
      </strong>
    </p>
  </div>,
  ";",
]
`;
